components/Login.js

```diff
@@ -53,61 +60,75 @@ export default class Login extends Component {
+                  console.log("userId:",response.data.principal.userId);
+                  console.log("organizationId:",response.data.principal.parentOrganizationId);
+                  console.log("parentOrganizationId:",response.data.principal.organizationId);
+                  console.log("organizationEmail:",responseAu.data.organizationEmail);
+                  console.log("organizationDisplayName,:",responseAu.data.organizationDisplayName);
+                  console.log("UserType:",responseUsr.data.type);
+                  console.log("liveTemplate",responseAu.data.liveTemplate);
+                  console.log("logourl",responseAu.data.logoUrl);
+                  console.log("appId",responseUsr.data.appId);
+                  console.log("authority:",response.data.accessRights);
+                  console.log("Inside login: end");
+                  //set session
+                  AsyncStorage.multiSet([
+                    ["userId",response.data.principal.userId],
+                    ["organizationId",response.data.principal.organizationId],
+                    ["parentOrganizationId",response.data.principal.parentOrganizationId],
+                    ["organizationEmail",responseAu.data.organizationEmail],
+                    ["organizationDisplayName",responseAu.data.organizationDisplayName],
+                    ["UserType",responseUsr.data.type],
+                    ["authority", response.data.accessRights],
+                    ["liveTemplate",responseAu.data.liveTemplate],
+                    ["logourl",responseAu.data.logoUrl],
+                    ["appId",responseUsr.data.appId],
+                    ["password",this.state.pass,],
+                    ["username",this.state.emailid],
+                    // ["signUpEnabled"],
+                    // ["homePageEnabled"],
+                    // ["telephone1",responseUsr.data.telephone1],
+              ]);
+              this.refs.Load.CloseLoad();
+            noBack(this.props,'Dash');
+            });
+            });
+            
+        }else{
           Alert.alert("Wrong username or password");
-          console.log("Something out of control");
-          console.log("you are logged in status:", responseobj.data.principal.authorities[0].authority);
-        }
-
-      })
-      .catch((error) => {
-        console.error(error);
-      });
+          this.refs.Load.CloseLoad();
+          }
+      }).catch((error) => {console.error(error);});
   }
 
-  //redirect function for future need
-  //reDirect(){
-  //  //body to reDirect
-  //}
 
   componentDidMount() {
     //internet connection check
     NetInfo.isConnected.fetch().then(isConnected => {
       console.log('First, is ' + (isConnected ? 'online' : 'offline'));
       if (isConnected) {
         this.setState({ btn: false })
-      }
-      else {
+      }else {
         Alert.alert("Net info alert", "Please connnect to internet for further app use");
```

Comment:
Use react-native-i18n to achieve Internationalization and move all the hard-coded string constants to translation file